app/lib/methods/audioPlayer.ts

```diff
@@ -14,17 +22,30 @@ const AUDIO_MODE = {
 	interruptionModeAndroid: InterruptionModeAndroid.DoNotMix
 };
 
+const getAudioKey = ({ msgId, rid, uri }: { msgId?: string; rid: string; uri: string }) => `${msgId}-${rid}-${uri}`;
+
 class AudioPlayer {
 	private audioQueue: { [audioKey: string]: Audio.Sound };
 	private audioPlaying: string;
+	private audiosRendered: Set<string>;
 
 	constructor() {
 		this.audioQueue = {};
 		this.audioPlaying = '';
+		this.audiosRendered = new Set<string>();
```

Comment:
This property holds the values of the audios displayed on the screen. Consequently, if there is another audio in sequence but it hasn't been rendered, the audio won't play automatically. This adjustment was made to address a scenario where you have two audios in the main room. If the last audio is preceded by another audio, located within a thread that you've opened before, the issue is now resolved.

Reply:
Great catch @reinaldonetof If you come across any issues with my contribution, please feel free to let me know. I'm always available and more than happy to contribute. Additionally, if it's convenient for you, ping me in official react native rc channel , that way I can clarify my doubts in real time, Thanks

Reply:
Hey @jsathu07 first of all, thanks a lot for your initiative and PR. 🚀 

Regarding the changes and some refactors that I made, don't worry about them. After we complete the internal review and QA process, I will discuss with you to merge these changes into your PR. Then, you will be able to merge it into develop.